pkg/requeue/requeue.go

```diff
@@ -44,6 +52,31 @@ func NeededAfter(
 	}
 }
 
+// An error to instruct the ACK runtime to process an error, but not requeue the
```

Comment:
docstring should begin "// NoRequeue is a struct that contains an error to instruct.."

Reply:
Copied from the error docstrings below, but I'll fix all of them 👍🏼